/*
 *  kdAmn is a deviantART Messaging Network client
 *  Copyright (C) 2013-2015  Cedric OCHS
 *
 *  This program is free software: you can redistribute it and/or modify
 *  it under the terms of the GNU General Public License as published by
 *  the Free Software Foundation, either version 3 of the License, or
 *  (at your option) any later version.
 *
 *  This program is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *  GNU General Public License for more details.
 *
 *  You should have received a copy of the GNU General Public License
 *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 *
 */

#ifndef CONNECTDIALOG_H
#define CONNECTDIALOG_H

#include "ui_connectdialog.h"

class ConnectDialog : public QDialog, public Ui::ConnectDialog
{
	Q_OBJECT

public:
	ConnectDialog(QWidget* parent);
	virtual ~ConnectDialog();

public Q_SLOTS:
	virtual void accept();
};

#endif
